packages/hardhat-core/src/internal/sentry/anonymizer.ts

```diff
@@ -93,6 +93,14 @@ export class Anonymizer {
   }
 
   public anonymizeErrorMessage(errorMessage: string): string {
+    const mnemonicPhraseLengthThreshold = 7;
+    const minimumAmountOfWordsToAnonymize = 4;
```

Comment:
These can be module-level constants, no need to parameterize a private function.

Reply:
By the way, I don't understand why we need two constants? It feels like this could be done with just one.

Reply:
We discussed this with @scnale in a call.

The first one is just an optimization, to avoid running the mnemonic detection algorithm on every string.

The second one is which is the minimum number of consecutive mnemonic words to clear. The reason for this is that we still want to anonymize most of it if there's a typo in one of its words. This way, we would anonymize `minimumAmountOfWordsToAnonymize+` words on each side of the typo.

The first one is just a performance optimization. The second one is actually how many words we anonymize at once. TH

Reply:
I agree about the module level constants